storage/cloud-client/hmac_samples_test.py

```diff
@@ -35,8 +36,7 @@
 SERVICE_ACCOUNT_EMAIL = os.environ["HMAC_KEY_TEST_SERVICE_ACCOUNT"]
 STORAGE_CLIENT = storage.Client(project=PROJECT_ID)
 
-
-@pytest.fixture
+@pytest.fixture(scope="module")
```

Comment:
Does this correct the fixture once for the entire test suite? If `test_delete_key` is ran first then it would be deleted for all.

Reply:
With the way we have things configured, within the module these will run in order. Yes, if delete is run first, the key will be gone for our tests. So this change makes the order significant.